packages/elements/src/list/elements/list.ts

```diff
@@ -654,6 +654,10 @@ export class List<T extends DataItem = ItemData> extends ControlElement {
       this.renderTimestamp.clear(); // force render of all items
       this.setAttribute('aria-multiselectable', this.multiple ? 'true' : 'false');
     }
+
+    if (changeProperties.has('stateless')) {
+      this.setAttribute('aria-readonly', this.stateless ? 'true' : 'false');
+    }
```

Comment:
I prefer to remove the `aria-readonly="false"` to make the html tag clean and simple.

```suggestion
    
    if (changeProperties.has('stateless')) {
      if (this.stateless) {
        this.setAttribute('aria-readonly', 'true');
      } else {
        this.removeAttribute('aria-readonly');
      }
    }
```